var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -127,14 +133,6 @@ def scalapack_libs(self):
         )
         return libs
 
-    def install(self, spec, prefix):
-        self.intel_prefix = os.path.join(prefix, "pkg")
-        IntelInstaller.install(self, spec, prefix)
-
-        mkl_dir = os.path.join(self.intel_prefix, "mkl")
-        for f in os.listdir(mkl_dir):
-            os.symlink(os.path.join(mkl_dir, f), os.path.join(self.prefix, f))
```

Comment:
you remove symlinks from all derived packages. I am ok with that, but we need to make sure it still works.